cell/errors.go

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cell
+
+type DuplicateFlagError struct {
+	flag string
+}
+
+func (dfe DuplicateFlagError) Error() string {
+	return "Duplicate flag '" + dfe.flag + "'"
```

Comment:
nit: error strings should not be captialized ([since it looks weird when wrapped](https://go.dev/wiki/CodeReviewComments#error-strings))